pyinterprod/cli.py

```diff
@@ -389,24 +389,14 @@ def run_member_db_update():
     if member_dbs or feature_dbs or site_dbs:
         tasks += [
             Task(
-                fn=interpro.iprscan.import_tables,
+                fn=interpro.iprscan.update_tables,
```

Comment:
We're not updating _tables_, we're importing data then updating partitions.

```suggestion
                fn=interpro.iprscan.import_matches_or_sites,
```